usr/lib/ep11_stdll/new_host.c

```diff
@@ -2513,17 +2515,27 @@ CK_RV SC_Decrypt(STDLL_TokData_t *tokdata, ST_SESSION_HANDLE *sSession,
                                     length_only, sess->decr_ctx.key,
                                     pEncryptedData, ulEncryptedDataLen,
                                     pData, pulDataLen);
-        if (!is_rsa_mechanism(sess->decr_ctx.mech.mechanism) && rc != CKR_OK)
+        mask = ~constant_time_is_zero(
```

Comment:
We could leave the original if statement in comments here, as you did it above. It makes reading the code easier.

Reply:
Ah yes. I did so in the other new_host files, but not here. Will fix.